### PR TITLE
Add do-cherry-picks entrypoint

### PR DESCRIPTION
**Pull request by petr-balogh**

### CODE REVIEW

**Review by dahorak — DISMISSED** [review]

File: ocs_ci/utility/scripts/cherry_pick/cherry-picks.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -e
+
+upstream=origin
+branch_name=$( git rev-parse --abbrev-ref HEAD )
+commit=$( git rev-parse HEAD )
+github_remote=$( git remote -v |grep -m1 pbalogh |sed 's/.*:\(.*\)\/.*/\1/' )
```
Comment (DanielOsypenko) [inline ocs_ci/utility/scripts/cherry_pick/cherry-picks.sh:7]: looks like the user "pbalogh" is hardcoded here

Comment (petr-balogh) [reply ocs_ci/utility/scripts/cherry_pick/cherry-picks.sh]: Good catch, it was supposed to be replaced by $FORK ;)

File: ocs_ci/utility/scripts/cherry_pick/cherry-picks.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -e
+
+upstream=origin
```
[the remark this answers is not in the document]
Comment (petr-balogh) [reply ocs_ci/utility/scripts/cherry_pick/cherry-picks.sh]: Fixed, and also provided new params fro branch name and specific comit.

**Comment by ebenahar** [comment]
> This pull request has been automatically marked as stale because it has not had recent activity. It will be closed in 30 days if no further activity occurs.

@petr-balogh should we proceed with this PR?

**Comment by prsurve** [comment]
@petr-balogh @ebenahar as we see Openshift Bot is added to our repo we can make use of that to create cherryypick pr

**Comment by petr-balogh** [comment]
experimenting with openshift-ci now and cherry-pick command, so this one I will close for now

**Pull request closed without merging**